config/attributes/clothing.json

```diff
@@ -1,33 +1,54 @@
 {
   "size": [
-    "0 (XXS)",
-    "2 (XS)",
-    "4 (S)",
-    "6 (S)",
-    "8 (M)",
-    "10 (M)",
-    "12 (L)",
-    "14 (L)",
-    "16 (XL)",
-    "18 (1X)",
-    "20 (2X)",
-    "22 (3X)"
+    "XXS",
```

Comment:
Would that create a new field (e.g. "shoe size") that appears when sellers
are listing items? I'm thinking that if, say, a seller is listing a piece
of furniture, having to go through and select "N/A" for each size field
would be cumbersome